metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/lineage.py

```diff
@@ -918,3 +926,178 @@ def test_capability(self, project_id: str) -> None:
     def report_status(self, step: str, status: bool) -> None:
         if self.redundant_run_skip_handler:
             self.redundant_run_skip_handler.report_current_run_status(step, status)
+
+    def gen_lineage_workunits_for_external_table(
+        self,
+        dataset_urn: str,
+        ddl: Optional[str],
+        graph: Optional[DataHubGraph] = None,
+    ) -> Iterable[MetadataWorkUnit]:
+
+        if not ddl:
+            return
+
+        # Expect URIs in `uris=[""]` format
+        uris_match = re.search(r"uris=\[([^\]]+)\]", ddl)
+        if not uris_match:
```

Comment:
If this should not happen then I think it worth to log it

Reply:
Yes, Added